### DISCUSSION TITLE
Webpack the Code of Babel + env-preset

### DISCUSSION THREAD

**Discussion by niklbird** [babel/babel#14196] [Q&A] [answered]
Hey!
I'm trying to use webpack to package the code of Babel into a single JS file. Packing Babel-Cli work, but I also need to use the environment preset and when I webpack the node_modules/babel-preset-env/lib/index.js, the resulting js file does not work to transpile ES6 to ES5. 

When I run babel with the generated index.js as a preset, it does not give any errors but the code in the output file is not changed.

My webpack config for the environment file looks like this:
const path = require('path');

module.exports = {
  entry: './node_modules/babel-preset-env/lib/index.js',
  output: {
    filename: 'index.js',
    path: path.resolve(__dirname, 'dist'),
  },
};

I run babel with node: 

node node_modules/babel-cli/bin/babel.js src/file.js -d dist --presets ./index.js
which produces: src/file.js -> dist/file.js

Any ideas / help would be greatly appreciated as I'm not very experienced with js-related development :)
Thank you alot!

**Comment by JLHwung** [comment, marked as answer]
Bundling `@babel/core` and other presets is not trivial: They are Node.js applications. I would suggest just use `@babel/standalone` for your purpose.

**Comment by niklbird** [comment]
I see, thank you, I will try that!
